GetPet/GetPet.Crawler/Utils/ParserUtils.cs

```diff
@@ -10,20 +11,19 @@ public static class ParserUtils
         private static List<string> _male = new List<string>() { "זכר", "male", "Male" };
         private static List<string> _female = new List<string>() { "נקבה", "female", "Female" };
 
-        public static string ConvertGender(string input)
+        public static Gender ConvertGender(string input)
```

Comment:
use contains with OrdinalIgnoreCase, u don't need to list all the cases in the list

Reply:
its irrelevant and probably will be remove because as of now its all in Hebrew (will remove when ill add more sites and verify all in indeed in hebrew)